packages/broker/src/ConfigWizard.ts

```diff
@@ -82,200 +84,164 @@ const DefaultConfig: Config = {
-        const valueString = await this.inquirerSinglePrompt(prompt)
-        const value = valueString === '' ? defaultValue: parseInt(valueString)
-
-        if (Number.isNaN(value) || !Number.isInteger(value)) {
-            logger.warn(`Non-numeric value [${valueString}] provided`)
-            return await this.promptNumberWithDefault(prompt, defaultValue)
-        }
-
-        if (value < 1024 || value > 49151) {
-            logger.warn(`Out of range port [${value}] provided (valid range 1024-49151)`)
-            return await this.promptNumberWithDefault(prompt, defaultValue)
-        }
-
-        return value
-    }
-
-    private async promptExistingPathWithDefault(prompt: inquirer.InputQuestion, defaultValue: string): Promise<string>{
-        const returnedValue = await this.inquirerSinglePrompt(prompt)
-        // defaultValue always exists at this point
-        if (returnedValue === '') {
-            return defaultValue
-        }
-
-        if (!existsSync(returnedValue)) {
-            logger.warn(`Path [${returnedValue}] does not exist`)
-            return await this.promptExistingPathWithDefault(prompt, defaultValue)
-        }
-
-        return returnedValue
-    }
-
-    async generateOrImportPrivateKey(): Promise<Wallet>{
-        const generateOrImport = await this.inquirerSinglePrompt({
-            type: 'list',
-            name:'generateOrImportEthereumPrivateKey',
-            message: 'Do you want to generate a new Ethereum private key or import an existing one?',
-            choices: ['Generate', 'Import'],
-            filter: (choice: string) => {
-                return choice.toLowerCase()
-            }
-        })
-
-        let wallet: Wallet
-        if (generateOrImport === 'generate') {
-            wallet = Wallet.createRandom()
-            this.config.ethereumPrivateKey = wallet.privateKey
-            logger.info('This is your generated private key. Please store it in a secure location')
-            logger.alert(this.config.ethereumPrivateKey)
-        } else if (generateOrImport === 'import') {
-            const privateKey = await this.inquirerSinglePrompt({
-                type: 'input',
-                name: 'privateKey',
-                message: "'Please provide the private key to import'",
-            })
-
+export let prompts: Array<inquirer.Question | inquirer.ListQuestion | inquirer.CheckboxQuestion> = [
+    {
+        type: 'list',
+        name:'generateOrImportEthereumPrivateKey',
+        message: 'Do you want to generate a new Ethereum private key or import an existing one?',
+        choices: ['Generate', 'Import']
+    },
+    {
+        type: 'input',
+        name:'importPrivateKey',
+        message: 'Please provide the private key to import',
+        when: (answers: inquirer.Answers) => {
+            return answers.generateOrImportEthereumPrivateKey === 'Import'
```

Comment:
'Generate' and 'Import' could be constants are we are referring to the value in multiple places